examples/orchestrator.yaml

```diff
@@ -44,3 +51,6 @@ spec:
       configMaps:
         - name: app-config-rhdh
     dynamicPluginsConfigMapName: orchestrator-plugin
+    extraEnvs:
+      secrets:
+        - name: backstage-backend-auth-secret # secret that contains the BACKEND_SECRET key
```

Comment:
To make this example usable OOTB, how about adding the expected Secret in this file as well? It shouldn't matter if it contains some dummy data, as this is just a full example.